aws/defaults/handlers.go

```diff
@@ -182,6 +183,35 @@ var ValidateResponseHandler = aws.NamedHandler{
 		}
 	}}
 
+// RequestInvocationIDHeaderHandler sets the invocation id header for request
+// tracking across attempts.
+var RequestInvocationIDHeaderHandler = aws.NamedHandler{
+	Name: "core.RequestInvocationIDHeaderHandler",
+	Fn: func(r *aws.Request) {
+		const invocationIDHeader = "amz-sdk-invocation-id"
+		r.HTTPRequest.Header.Set(invocationIDHeader, r.InvocationID)
```

Comment:
scope of `const invocationIDHeader` will be limited to the named Handler func; Is that intended/needed?

Reply:
Yeah, the value didn't really need to be a constant, and could of been in-lined. Didn't want it to be exported.